runtime/cudaq/platform/mqpu/nvqc.config

```diff
@@ -1,52 +0,0 @@
-# ============================================================================ #
-# Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                   #
-# All rights reserved.                                                         #
-#                                                                              #
-# This source code and the accompanying materials are made available under     #
-# the terms of the Apache License 2.0 which accompanies this distribution.     #
-# ============================================================================ #
-
-msg=""
-
-# Tell NVQ++ to generate glue code to set the target backend name
-GEN_TARGET_BACKEND=true
-
-# Use the MultiQPUPlatform
-PLATFORM_LIBRARY=mqpu
-
-# QPU subtype
-PLATFORM_QPU=NvcfSimulatorQPU
-
-# Disable library mode
-LIBRARY_MODE=false
-
-LINKLIBS="${LINKLIBS} -lcudaq-remote-simulator-qpu" 
-
-PLATFORM_EXTRA_ARGS=""
-# NB: extra arguments always take the form:
-#   --<target>-<option> <value>
-# as in
-#   --remote-sim-backend foo
-while [ $# -gt 1 ]; do
-	case "$1" in
-	--nvqc-backend)
-		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;backend;$2"
-		;;
-	--nvqc-nqpus)
-		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;nqpus;$2"
-		;;
-	--nvqc-ngpus)
-		PLATFORM_EXTRA_ARGS="$PLATFORM_EXTRA_ARGS;ngpus;$2"
-		;;
-	--nvqc-function-id)
```

Comment:
With the changes, I don't see where this could be set now. Is this still something that can be configured on the command line? Or is this now something that can only be configured via environment variable? Same question for the rest of the arguments here.

Reply:
The number of QPUs, GPUs, and backend can still be specified (via `--nvqc-nqpus`, `--nvqc-ngpus`, and `--target`, respectively). You are correct that I removed the ability to set the function id, function version id, and API key via command line. They will only be able to be overridden via environment variables. Most users will never have the need to override the function id or function version id, and re: api key, I think it is better if we don't embed that in the executable like this option was previously allowing.